cep-??.md

```diff
@@ -0,0 +1,134 @@
+<table>
+<tr><td> Title </td><td> Specification of <code>environment.yml</code> input files </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> June 4, 2024 </td></tr>
+<tr><td> Updated </td><td> June 4, 2024 </td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/81 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CEP standardizes the format of `environment.yml` input files for conda clients.
+
+## Motivation
+
+The motivation of this CEP is merely informative. It describes the details of an existing file format.
+
+## Nomenclature
+
+The `environment.yml` file format was introduced by `conda env`. It has been traditionally referred to as "the YAML input file", but other YAML files exist in the ecosystem now. To avoid confusion, in this CEP we will only use the term `environment.yml`. Do note that the actual filename of an input file using this format might be a different one (e.g. `env.yml`).
+
+## Specification
+
+`environment.yml` files are YAML documents that encode the necessary information to create a new conda environment. They can contain up to five top-level keys:
+
+- `name`: The preferred name for the environment.
+- `prefix`: The preferred full path to the environment. Often ignored.
+- `dependencies`: List of package specifications that MUST be installed in the new environment.
+- `channels`: conda channels that will be used to resolve the dependencies.
+- `variables`: Environment variables that SHOULD be added to the `conda-meta/state` file in the resulting environment.
+
+Additional sections can be present, but they MUST be ignored and the user SHOULD receive an informative warning about them.
```

Comment:
conda-lock cannot abide by this CEP because it includes a `platforms` key and one other. Any ideas on how to make that work?